### DISCUSSION TITLE
Multiselect combobox release enquiry

### DISCUSSION THREAD

**Discussion by ghost** [twilio-labs/paste#1805] [General] [answered]
Hey Team,

We are working on SSO product where user needs to select multiple domains from available verified domain list (It can be 100 or more). We believe multi-select combobox is perfect for this use case .

As far as we know, Mutli-Select Combobox release is planned by end of August. So wanted to check with you about exact date of release and when can we expect to use it in our product?

**Comment by TheSisb** [comment, marked as answer]
Hi @Kajal-Joshi,

As per the [roadmap](https://paste.twilio.design/roadmap), the [multiselect primitive has been released](https://paste.twilio.design/primitives/combobox-primitive#multiselect-combobox-example) as of today. You can [learn more about Paste primitives on this page](https://paste.twilio.design/tokens/how-to-compose-custom-ui-with-tokens/#design-primitive-components).
